modules/nf-core/vcf2maf/tests/nextflow.config

```diff
@@ -0,0 +1,12 @@
+// config for running without VEP
+params.species = "homo_sapiens"
+params.build   = "GRCh38"
+
+process {
+            publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+            withName: 'VCF2MAF' {
+                ext.args = "--ncbi-build ${params.build} --species ${params.species} --inhibit-vep"
+                // NOTE: using a different container here than in the main.nf for VCF2MAF
```

Comment:
Maybe just add a note in the pull request documenting why a different container was used for the test.   Was there an error with the other one?  Its fine to do this IMO but just specify why somewhere.